5._Governance.md

```diff
@@ -6,57 +6,55 @@ As a community effort organized within the [Supply Chain Integrity WG](https://g
+**1.4. Steering Committee Members.** The "Steering Committee" is the body that is responsible for overseeing the overall activities of the Project. The Steering Committee consists of 5 Participants (each, a "Steering Committee Member") and will initially consist of the Steering Committee Members so designated as of the date of initial adoption of this SLSA Governance Policy. The Steering Committee will meet regularly as needed, but no less then once per quarter. Examples of the responsibilities of the Steering Committee include:
 
 * enabling the smooth running of the Project
-* coordinating activities between workstreams and Maintainers
-* collectively reviewing and revising the roadmap on a biannual basis
+* collectively reviewing and revising the Project mission, vision, strategy, and roadmap on a biannual basis
+* publishing quarterly updates suitable for OpenSSF TAC and other working groups' consumption
 * participating in strategic planning, such as coordinating face-to-face meetings or suggesting and approving changes to the governance model
-* creating or restructuring workstreams
+* managing the lifecycle of existing and proposed Work Streams within the Project with the consensus input from the community of Participants
+* creating or restructuring Work Streams
 * responding to specific issues or concerns above and beyond the domain of the various workstreams
 * making decisions when community consensus cannot be reached, pursuant to the appeal process documented below
+* holding Maintainers accountable to the standards of what makes SLSA an effective Project and representative of a diverse community of Participants and Contributors. In the extreme event that a majority of existing Maintainers are unavailable to approve additions or removals for an extended period of time, the Steering Committee may add or remove Maintainers to bring the Project back to a functioning state
 
-**1.5. Steering Committee Member Terms.** During the initial year, the Steering Committee Members will agree on grouping themselves into two groups, one to serve an initial two-year term, and the other for an initial one-year term. Thereafter, the Steering Committee Members will serve two year terms.
-
-At the expiration of a Steering Committee Member term, any Participant may submit a nomination to fill the seat. An individual may be nominated for and serve any number of successive terms.
-
-If a Steering Committee Member resigns or ceases to participate for a significant period of time prior to the end of their term, the remaining Steering Committee Members may choose to remove that Steering Committee Member. If so, the remaining Steering Committee Members will determine whether and when to fill the role.
 
-The Steering Committee may add additional Steering Committee Members as it deems necessary.
+**1.5. Steering Committee Member Terms.** Steering Committee Members have a one year term.
```

Comment:
Does this set us up for all the seats expiring at one time and then there's no steering committee members?

Perhaps the old language had that problem too?

Reply:
We propose to not stagger elections like in the current governance since that adds more overhead. To your point, we should note that nominations & elections should occur before the term is up, maybe a month in advance to give enough time.

Reply:
Added a line about starting this one month in advance.

Reply:
Technically it is possible for all current members to "expire" and for none of them to self-nominate to serve again. However, this is not meaningfully different than if we had a 2 year term and someone basically walked away for whatever reason. This is simple enough and it is reasonable that SC members will serve multiple terms in succession or naturally find their replacements. In the event that literally nobody is nominated (or self-nominates) for the role then the project larger issues.